libs/features/commerce/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@milo/commerce",
+  "module": "./src/index.js",
+  "type": "module",
+  "types": "./src/index.d.ts",
+  "version": "0.0.1",
+  "scripts": {
+    "build": "npm run build:bundle && npm run build:types",
+    "build:bundle": "esbuild index.js --alias:react=./src/react-shim.js --bundle --format=esm --metafile=stats.json --minify --outfile=../../deps/commerce.js --platform=browser --sourcemap --target=es2020",
+    "build:types": "cp types.d.ts ../../deps/commerce.d.ts",
+    "build:watch": "npm run build:bundle --watch",
+    "test": "wtr --config ../../../web-test-runner.config.mjs \"./test/**/*.test.(js|html)\" --node-resolve --port=2001 --coverage --concurrent-browsers 4",
+    "test:watch": "npm test -- --watch",
+    "test:file": "wtr --config ../../../web-test-runner.config.mjs --coverage --node-resolve --port=2001",
+    "test:file:watch": "wtr --config ../../../web-test-runner.config.mjs --coverage --node-resolve --port=2001 --watch"
+  },
+  "dependencies": {
+    "@dexter/tacocat-consonant-templates": "^1.9.7",
+    "@dexter/tacocat-core": "^1.10.3",
```

Comment:
@npeltier, I agree that this would simplify Milo Commerce deps, however `tacocat-core` package was made free of any dependencies by design
instead, I'd recommend to create a new, specially crafted tacocat npm package for Milo, give it a self-explanatory name like `@dexter/tacocat-milo` and reexport all necessary dependencies from there
this could be done in a separate PR to prevent this one from bloating more